src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -597,6 +598,7 @@ else if (world.getEnvironment() == Environment.NORMAL)
         this.config_limitTreeGrowth = config.getBoolean("GriefPrevention.LimitTreeGrowth", false);
         this.config_pistonExplosionSound = config.getBoolean("GriefPrevention.PistonExplosionSound", true);
         this.config_pistonMovement = PistonMode.of(config.getString("GriefPrevention.PistonMovement", "CLAIMS_ONLY"));
+        this.config_pistonAllowIntersectionTrustedClaims = config.getBoolean("GriefPrevention.Piston", true);
```

Comment:
Assuming this is a missed string - `GriefPrevention.Piston` is terribly undescriptive.

Reply:
Oh, yea it is.